.github/workflows/release_mac.yml

```diff
@@ -0,0 +1,70 @@
+name: MacRelease
+
+on:
+  push:
+    branches:
+      - rel-*
+  # test this PR for now, remove it before the merge
+  pull_request:
+    branches: [ master ]
+env:
+  MACOSX_DEPLOYMENT_TARGET: 10.12
+jobs:
+  build:
+    runs-on: macos-10.15
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+        architecture: ['x64']
```

Comment:
for mac x86_64 is more appropriate

Reply:
However, GitHub Action does not support x86_64 Python so I keep using x64 here.